src/php/aspell.php

```diff
@@ -60,6 +60,12 @@
     // when splitting words.
     $text = preg_replace('/--+/u', ' ', $text);
 
+    // Get rid of e-mail addresses
+    $text = preg_replace("/[^@\s]*@[^@\s]*\.[^@\s]*/", ' ', $text);
+
+    // Get rid of URLs
+    $text = preg_replace("/[a-zA-Z]*[:\/\/]*[A-Za-z0-9\-_]+\.+[A-Za-z0-9\.\/%&=\?\-_]+/i", ' ', $text);
```

Comment:
I am not sure this is a good RegEx to use here, because it is too broad. It will cover cases like this:

> something...misspelled

and this:

> no space.after a dot